site/how-it-works/encryption.md

```diff
@@ -32,6 +33,11 @@ extensions to TLS such as [DTLS](https://tools.ietf.org/html/rfc4347)
 which can operate over UDP, these are not widely implemented and
 deployed.
 
+In the case of fast datapath, UDP packets are encrypted by using
```

Comment:
I would keep it as it is, as the beginning of this page talks about control plane and data plane (sleeve) traffic encryption which uses the same primitives.